tests/unit/test_common.py

```diff
@@ -32,6 +32,120 @@ def test_yaml_variables_will_result_in_equivalent_output_to_json_file(self):
         yaml_default_env = get_deployment_config(yaml_file).get_environment("default")
         assert yaml_default_env == json_default_env
 
+    def test_json_file_with_env_variables_scalar_type(self):
+        """
+        JSON: Simple Scalar (key-value) type for timeout_seconds parameter
+        """
+        # Set TIMEOUT env var
+        os.environ["TIMEOUT"] = "100"
```

Comment:
Please use patch approach as described [here](https://adamj.eu/tech/2020/10/13/how-to-mock-environment-variables-with-pytest/), the approach with `os.environ` is not thread safe